packages/console/src/pages/Connectors/components/Guide/Form.tsx

```diff
@@ -0,0 +1,92 @@
+import type { ConnectorFactoryResponse } from '@logto/schemas';
+import { useState } from 'react';
+import { Controller, useFormContext } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+
+import Button from '@/components/Button';
+import CodeEditor from '@/components/CodeEditor';
+import FormField from '@/components/FormField';
+import TextInput from '@/components/TextInput';
+
+import * as styles from './index.module.scss';
+import type { CreateConnectorForm } from './types';
+
+type Props = {
+  connector: ConnectorFactoryResponse;
+};
+
+const Form = ({ connector }: Props) => {
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+  const { configTemplate, isStandard } = connector;
+  const { control, register } = useFormContext<CreateConnectorForm>();
+  const [darkVisible, setDarkVisible] = useState(false);
+
+  const toggleDarkVisible = () => {
+    setDarkVisible((previous) => !previous);
+  };
+
+  return (
+    <div>
+      {isStandard && (
+        <>
+          <FormField isRequired title="connectors.guide.name">
+            <TextInput
+              placeholder={t('connectors.guide.name')}
+              {...register('name', { required: true })}
+            />
+            <div className={styles.tip}>{t('connectors.guide.name_tip')}</div>
```

Comment:
This looks like a tooltip; maybe sync this information with the designer.

Reply:
Per discussion, a design system project will cover this.